To: vim_dev@googlegroups.com
Subject: Patch 7.3.424
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.424
Problem:    Win16 version missing some functions.
Solution:   Add #defines for the functions.
Files:      src/gui_w16.c


*** ../vim-7.3.423/src/gui_w16.c	2011-01-17 20:08:03.000000000 +0100
--- src/gui_w16.c	2011-08-10 16:50:21.000000000 +0200
***************
*** 21,26 ****
--- 21,32 ----
   *
   */
  
+ /* Win16 doesn't use the "W" methods. */
+ #define pDispatchMessage DispatchMessage
+ #define pGetMessage GetMessage
+ #define pIsDialogMessage IsDialogMessage
+ #define pPeekMessage PeekMessage
+ 
  /*
   * Include the common stuff for MS-Windows GUI.
   */
*** ../vim-7.3.423/src/version.c	2012-02-04 21:57:44.000000000 +0100
--- src/version.c	2012-02-04 22:01:13.000000000 +0100
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     424,
  /**/

-- 
Now it is such a bizarrely improbable coincidence that anything as
mind-bogglingly useful as the Babel fish could have evolved purely by chance
that some thinkers have chosen to see it as a final and clinching proof of the
NON-existence of God.
The argument goes something like this: 'I refuse to prove that I exist,' says
God, 'for proof denies faith, and without faith I am nothing.'
'But,' says Man, 'the Babel fish is a dead giveaway, isn't it?  It could not
have evolved by chance.  It proves you exist, and so therefore, by your own
arguments, you don't.  QED.'
'Oh dear,' says God, 'I hadn't thought of that,' and promptly vanishes in a
puff of logic.
'Oh, that was easy,' says Man, and for an encore goes on to prove that black
is white and gets himself killed on the next pedestrian crossing.
		-- Douglas Adams, "The Hitchhiker's Guide to the Galaxy"

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
